ksql-engine/src/main/java/io/confluent/ksql/services/DefaultConnectClient.java

```diff
@@ -214,6 +224,33 @@ public DefaultConnectClient(
     }
   }
 
+  @Override
+  public ConnectResponse<Map<String, Map<String, List<String>>>> topics(final String connector) {
```

Comment:
The reason is that the key is the actual name of the connector and not `connector : <connector name>`

Maybe my json foo is missing a way to deserialize this with a `JsonCreator` and the constructor, but based on what I know it can't be done with jackson and the return type that you mention (which is a bummer but we use that notation in other endpoints already). The return type is a bit verbose, indeed.